websites/views.py

```diff
@@ -190,7 +190,7 @@ def get_serializer(self, *args, **kwargs):
             and self.request.data.get("title")
         ):
             request_data = self.request.data.copy()
-            request_data["name"] = slugify(request_data["title"])
+            request_data["name"] = slugify(request_data["title"], allow_unicode=True)
```

Comment:
Unicode `name`s don't seem to work for me in Studio. Seems like a frontend issue, because I see no request being made to fetch data from the backend.

<img width="1785" alt="Screenshot 2024-02-07 at 5 26 49 PM" src="https://github.com/mitodl/ocw-studio/assets/71316217/3ebba2ff-6cb2-4c08-94e5-a0bd7b283e88">